native/cocos/editor-support/spine/Constraint.h

```diff
@@ -43,8 +43,6 @@ class SP_API Constraint : public Updatable {
 
     virtual ~Constraint();
 
-    virtual void update() = 0;
```

Comment:
update function was already defined in its parent class `Updatable`. No need to define it here again.